lua/cmp_luasnip/choice.lua

```diff
@@ -0,0 +1,45 @@
+local cmp = require("cmp")
+
+local source = {}
+
+source.new = function()
+	return setmetatable({}, { __index = source })
+end
+
+source.get_keyword_pattern = function()
+	return "\\%([^[:alnum:][:blank:]]\\|\\w\\+\\)"
```

Comment:
I think this should be changed to trigger at any position I think. I tried it in the javadoc-snippet and `cmp` doesn't always show the choices.